content/information-distribution/node.md

```diff
@@ -44,81 +44,158 @@ This ping **must** take the form of a Matrix timeline event of the
 
 An entry node is a node that allows client implementations to connect to the
 federations this node is in, and retrieve information from them. An entry node
-**must** allow guest access, and **must** define at least one
-[alias](https://matrix.org/docs/spec/client_server/r0.4.0.html#room-aliases) to
-the Matrix room.
-
-Client implementations **must** provide users with an updated list of entry
-nodes with their addresses and the aliases to use to join the federation the
-node is in. Clients implementations **must** also provide users with a way to
-input custom values for the address of the entry node to use and the alias of
-the Matrix room to join.
-
-Once a Matrix room is joined, client implementations **must** retrieve and save
-the [list of
-aliases](https://matrix.org/docs/spec/client_server/r0.4.0.html#m-room-aliases)
-for this Matrix room. This can be done by processing all of the `m.room.aliases`
-events in the state of the Matrix room, each of them containing the list of
-aliases defined for a given node. The content for such an event looks like this:
+**must** allow guest access.
+
+On top of that, an entry node **must** expose a file at the address
+`https://SERVER_NAME/.well-known/informo/info` which content is a map in the
+JSON format, which associates Matrix room IDs with a map implementing the
+following structure:
+
+| Parameter    | Type   | Req. | Description                                                   |
+|:-------------|:-------|:----:|:--------------------------------------------------------------|
+| `entry_node` | `bool` |  x   | Whether the node is an entry node for this federation (room). |
+
+One entry of this map **can** be the wildcard value "`*`" instead of a Matrix
```

Comment:
Another way to see it is that the `*` object is the default value for informo federation objects, and the other objects overrides this default value on a per-field basis. 

The current sentense is OK for me, but you can change it if you think it can be better formulated.